misc/test/examples_test.go

```diff
@@ -227,8 +227,25 @@ func TestAccAwsGoAppSync(t *testing.T) {
 	test := getAWSBase(t).
 		With(integration.ProgramTestOptions{
 			Dir: path.Join(getCwd(t), "..", "..", "aws-go-appsync"),
-		})
+			ExtraRuntimeValidation: func(t *testing.T, stack integration.RuntimeValidationStackInfo) {
+				maxWait := 15 * time.Minute
 
+				endpoint := stack.Outputs["endpoint"].(string)
+				queryParams := "?query=mutation%20AddTenant%20%7B%20addTenant%28id%3A%20%22123%22%2C%20name%3A%20%22FirstCorp%22%29%20%7B%20id%20name%20%7D%20%7D"
+
+				key := stack.Outputs["key"].(string)
+				headersMap := map[string]string{
+					"Content-Type": "application/graphql",
+					"x-api-key":    key,
+				}
+
+				assertHTTPResultShapeWithRetry(t, endpoint+queryParams, headersMap, maxWait, func(body string) bool {
```

Comment:
The idiomatic way to concat strings in go is using the `fmt` package (Sprintf in this case). 

Something like:

```go
endpoint := stack.Outputs["endpoint"].(string)
mutation := "mutation AddTenant { addTenant(id: \"123\", name: \"FirstCorp\") { id name } }"
getURL := fmt.Sprintf("%s?query=%s", endpoint, url.QueryEscape(mutation))

assertHTTPResultShapeWithRetry(t, getUrl, ...
```